app/config/config_dev.yml

```diff
@@ -8,18 +8,18 @@ csa_guzzle:
 framework:
     profiler: ~
     router:
-        resource: '%kernel.root_dir%/config/routing_dev.yml'
+        resource: '%kernel.project_dir%/app/config/routing_dev.yml'
 
 monolog:
     handlers:
         console:
             channels:
-              - !event
+              - '!event'
             type: console
             formatter: monolog.formatter.line
         main:
             channels:
-              - !event
+              - '!event'
```

Comment:
values in `parameters.yml` are inserted by the formula without quotes: https://github.com/elifesciences/journal-formula/blob/master/salt/journal/config/srv-journal-app-config-parameters.yml#L25
I've seen values like https://github.com/elifesciences/journal-formula/blob/master/salt/pillar/journal.sls#L3 giving deprecation notices when they end up there.